sugoi-api-rest-old-services/src/main/java/fr/insee/sugoi/old/services/controller/IdController.java

```diff
@@ -94,7 +95,7 @@ private Contact findContact(Realm realm, String id) {
     try {
       return ouganextSugoiMapper.serializeToOuganext(
           userService.findById(realm.getName(), null, id), Contact.class);
-    } catch (EntityNotFoundException e) {
+    } catch (UserNotFoundException e) {
```

Comment:
findById doesn't throw UserNotFoundException does it ? here we should remove the try and check if we have something in the optional